src/main/java/io/github/fusionflux/portalcubed/PortalCubed.java

```diff
@@ -51,6 +56,11 @@ public void onInitialize(ModContainer mod) {
 
 		ConstructManager.init();
 		HoldableEntity.registerEventListeners();
+		DisintegrationSoundType.init();
+		EntityTrackingEvents.AFTER_START_TRACKING.register((tracked, player) -> {
```

Comment:
I don't like putting this here

Reply:
I could put in a static method in `EntityExt` but I would like it to be renamed if so, see my reply to `I don't like storing these here` in `EntityExt`

Reply:
there's not really anywhere better so I guess it's fine